src/naksu/mebroutines/install/install.go

```diff
@@ -144,10 +145,16 @@ func downloadFile(url string, filepath string) error {
 	return nil
 }
 
-// TestHTTPGet tests whether HTTP get succeeds to given URL
-func TestHTTPGet(url string) bool {
+// TestHTTPGet tests whether HTTP get succeeds to given URL in given timeout (seconds)
+func TestHTTPGet(url string, timeout int) bool {
```

Comment:
I think the public function can be opinionated here about the URL and timeout. I'd add `CheckIfNetworkAvailable()` function here, which internally uses the sane values for timeout and URL.

That way caller does not need to figure out the URL and timeout in when calling.

Reply:
Done - moved to separate package since this is feature does not have anything to do with installation.